src/cmd_line/subparsers/sort.ts

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+import * as node from "../commands/sort";
+
+export function parseSortCommandArgs(args : string) : node.SortCommand {
+
+  const reverse = args !== null && args.indexOf("!") >= 0;
+
+  return new node.SortCommand({
+    reverse: reverse,
```

Comment:
Fun fact: `{ reverse: reverse }` can be better written in ES6 as `{ reverse }`. :)